core/rio/turtle/src/main/java/org/eclipse/rdf4j/rio/turtle/Chars.java

```diff
@@ -0,0 +1,30 @@
+package org.eclipse.rdf4j.rio.turtle;
```

Comment:
True, thought that the maven build process is adding one for me. I will remove the file anyways.